### ISSUE TITLE
Our Voice USA mobile app and website

### ISSUE THREAD

**Issue by cormander** [ProgressiveCoders/projects#143]
### About your project		
We're building tools to make civic education and engagement easier. Our first project, a mobile app to show who represents you - and how to contact them - is now available in Android and iPhone stores. The app also has an unfinished canvassing tool - it's not in the release due to it being incomplete. Lots that can be done with that moving forward; 1) linking to people running against the incumbent, 2) linking in how a politician voted on bills, 2) listing the issues so the voters can define how they feel and then see how their views align with those who are running for office who also took the survey.

We'd also like to port the app to a website, using the same backend API. Might even be able to use a react native module to publish it to the web with some slight modifications, without having to rewrite anything.

### Who you are looking for		
While we have progressives in the organization, we're ultimately non-partisan and are making tools available that anyone can use. We believe that a more educated public, no matter what part of the political spectrum, will elect a more representative and less polarized public officials. By giving free tools to the hands of the grassroots, the cost to run for office goes down which enables more people to run successful campaigns with less money.

Bottom line; you have to be OK with the idea of someone who doesn't agree with you politically being able to use the tools you write.

### Tools		
React Native
NodeJS
Redis

### Contact Info
github.com/cormander
tech@ourvoiceusa.org

### Relevant Links 

https://github.com/OurVoiceUSA/
https://ourvoiceusa.org/our-voice-app/

**Comment by stephenscapelliti** [comment]
This is a great app. Excellent display and functionality.
Please join us in ProgCode: https://airtable.com/shraBEItZa0sYiMB0
